### DISCUSSION TITLE
Display more than one Relationship data with formula

### DISCUSSION THREAD

**Discussion by Winniepoom** [Budibase/budibase#11503] [Help] [answered]
Hello I'm trying to display data from relationships data by using formula, and it working well with index 0 but when I try to update index to be 1 or another number it showing me that my "Current Handlebars syntax is invalid"
![image](https://github.com/Budibase/budibase/assets/107398226/b0b544f1-ecc2-49a9-8ccb-cc5c5ba4739c)

My Goal :
in my id_issue column some row had more than one data, So I want my redmine_data can display like that but it seem like I can't update index in formula.
![tempsnip](https://github.com/Budibase/budibase/assets/107398226/58c787f2-4ec9-4931-a772-cbd701f806e7)

**Comment by ConorWebb96** [comment, marked as answer]
Hey @Winniepoom,

I think this is an issue relating to handle bars you can do this with JS; `return $("Jobs.0.Job ID") + ' ' + $("Jobs.1.Job ID")`.
<img width="572" alt="Screenshot 2023-08-11 at 10 41 34" src="https://github.com/Budibase/budibase/assets/126772285/8aa94b65-713f-4a24-944f-3ed7e95dba73">

I do feel like a more programmable way of doing this would be better. So I'm going to show you the below, it would avoid you having to endless update the formula and return 1,2,3,4,5 relationships as you add more.

```
const jobs = $("Jobs");
var emptyString = "";

if(!jobs) {
  return null;
}

for (const item of jobs) {
  emptyString += item._id + " ";
}

return emptyString;
```

<img width="272" alt="Screenshot 2023-08-11 at 10 57 53" src="https://github.com/Budibase/budibase/assets/126772285/abc1ac69-57cf-471a-bff2-9628f7e02a4f">

You should be able to modify the above to for your field names etc. But this should look through all objects within the relationship and output all the _ids separated by a space.

I hope this helps.

**Comment by Winniepoom** [comment]
Thank you @ConorWebb96 that solve my issue.
here my updated code.
```
var issueID1 = $("id_issue.0.issue_id");
var issueID2 = $("id_issue.1.issue_id");
var issueID3 = $("id_issue.2.issue_id");

if (issueID1 === undefined || issueID2 === undefined || issueID3 === undefined) {
    return "";
}

return issueID1 + ' ' + issueID2 + ' ' + issueID3;
```
